__tests__/app.test.js

```diff
@@ -60,6 +60,44 @@ describe("/api/reviews", () => {
   });
 });
 
+describe("/api/review/:review_id", () => {
+  test("GET - 200: returns the single specified review", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.review.length).toBe(1);
+        expect(body.review[0]).toMatchObject({
+          review_id: expect.any(Number),
```

Comment:
Since we know which review_id we are expecting back we should be checking for that specific one instead of any(Number).

You can keep using the any() matcher for the other keys though to make the test more flexible for later.